internal/pkg/cli/svc_deploy.go

```diff
@@ -396,7 +396,7 @@ func (o *deploySvcOpts) dfBuildArgs(svc interface{}) (*dockerengine.BuildArgumen
 func buildArgs(name, imageTag, copilotDir string, unmarshaledManifest interface{}) (*dockerengine.BuildArguments, error) {
 	type dfArgs interface {
 		BuildArgs(rootDirectory string) *manifest.DockerBuildArgs
-		TaskPlatform() (*string, error)
+		TaskPlatform() *string
```

Comment:
What do you think of 
```suggestion
		TaskPlatform() string
```
instead `""` means use whatever is Docker's default. The clients anyway have to do `aws.StringValue` afterwards

Reply:
I wonder if we should also rename the method to be `ContainerPlatform` instead to denote that this is related to building the container image and not the ECS platform. (For example, this method name is odd for a RDWS)